src/main/java/pages/LandingPage.java

```diff
@@ -6,7 +6,7 @@
 
 public class LandingPage {
```

Comment:
Each of page (login, myaccount page etc) could be called LandingPage. It should be changed to Homepage.